internal/store/common.go

```diff
@@ -152,6 +156,66 @@ func GetNonNilFieldsFromResource(resource model.Resource) []string {
 	return ret
 }
 
+func matchExpressionQueryAndArgs(matchExpression api.MatchExpression, colName string) (string, []any) {
+	params := strings.Join(lo.Map(lo.FromPtr(matchExpression.Values), func(_ string, _ int) string { return "?" }), ",")
```

Comment:
We might consider simply this and loop once something like:
```
params := make([]string, len(*matchExpression.Values))
args := make([]any, len(*matchExpression.Values))

for i, val := range *matchExpression.Values {
    params[i] = "?"
    args[i] = matchExpression.Key + "=" + val
}

return fmt.Sprintf("%s && ARRAY[%s]", colName, strings.Join(params, ",")), args
```

Reply:
OK